test/units/testsuite-02.service

```diff
@@ -4,5 +4,5 @@ After=multi-user.target
 
 [Service]
 ExecStartPre=rm -f /failed /testok
-ExecStart=sh -x -c 'systemctl --state=failed --no-legend --no-pager >/failed ; echo OK > /testok'
+ExecStart=sh -x -e -c 'mountpoint /var; systemctl --state=failed --no-legend --no-pager > /failed; echo OK > /testok'
```

Comment:
Actually, I'd prefer to get rid of the spaces between the redirection sign and file name. They make things longer and split what is essentially one thing into two.

Reply:
Will do. I just wanted to make them consistent, don't really have a preference in either one.